apps/backend/src/app/api/latest/contact-channels/verify/verification-code-handler.tsx

```diff
@@ -33,6 +34,21 @@ export const contactChannelVerificationCodeHandler = createVerificationCodeHandl
   async send(codeObj, createOptions, sendOptions: { user: UsersCrud["Admin"]["Read"] }) {
     const tenancy = await getSoleTenancyFromProjectBranch(createOptions.project.id, createOptions.branchId);
 
+    const contactChannel = await prismaClient.contactChannel.findUnique({
```

Comment:
can you add a comment explaining why and when this happens, and E2E tests to test it?